pkg/storage/mysql/mysqlstore_test.go

```diff
@@ -12,7 +12,9 @@ import (
 	"os"
 	"strings"
 	"testing"
+	"time"
 
+	"github.com/cenkalti/backoff"
```

Comment:
should be using `github.com/cenkalti/backoff/v4` - this version has go module support.

Reply:
@troyronda updated